src/xcode/ENA/ENA/Source/Protocols/CollectionViewCellConfigurator.swift

```diff
@@ -44,4 +44,8 @@ extension CollectionViewCellConfigurator {
 			fatalError(error)
 		}
 	}
+
+	var hashValue: AnyHashable {
```

Comment:
What does that even mean here?

Reply:
We use this value to get `Hashable` value of class which conforms `CollectionViewCellConfigurator`, for example `HomeActivateCellConfigurator`, `HomeInfoCellConfigurator`, .... We use this property in `applySnapshotFromSections` in HomeViewController.swift for DifferableDataSource api.  I did it as protocol extension, for comfortable using it.